src/edu/psu/sweng500/team8/coreDataStructures/SavePackage.java

```diff
@@ -0,0 +1,30 @@
+package edu.psu.sweng500.team8.coreDataStructures;
+
+import java.io.Serializable;
+
+import javax.swing.JTextField;
+
+public class SavePackage  implements Serializable {
+	private CellGrid m_cellGrid;

```

Comment:
The java convention does not use m_ . I used it out of habit from c++ but later learned this is not the standard